### PR TITLE
Improve the existing contribution guide

### PR DESCRIPTION
**Pull request by johnlk**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description
We didn't really have a contribution guide, so I'm adding one based on issue #68. 

## Notes to Reviewer
N/A

## How to test
N/A

## Link to issues addressed
N/A

### CODE REVIEW

File: .github/CONTRIBUTIONS.md [hunk 1]
```diff
@@ -1,13 +1,58 @@
...
+
+To contribute to this project, please follow these steps:
+
+1. Fork the repository and create a new branch for your feature or bug fix.
+2. Make your changes in the new branch, ensuring that your code follows the project's coding style and conventions.
+3. Write clear and concise commit messages that describe the purpose of your changes.
+4. Push your changes to your forked repository.
+5. Submit a pull request to the main repository, clearly explaining the purpose and scope of your changes by populating the pull request template.
+6. Update the unit tests if the change requires testing, ie not a documentation change.
+7. Engage in the discussion and address any feedback or questions raised during the code review process.
```
Comment (JavierCane) [inline .github/CONTRIBUTIONS.md:54]: Non-blocking suggestion: Reorder steps suggesting to test before opening the PR

```suggestion
1. Fork the repository and create a new branch for your feature or bug fix.
2. Make your changes in the new branch, ensuring that your code follows the project's coding style and conventions.
3. Write clear and concise commit messages that describe the purpose of your changes.
4. Update the unit tests if the change requires testing, ie not a documentation change.
5. Push your changes to your forked repository.
6. Submit a pull request to the main repository, clearly explaining the purpose and scope of your changes by populating the pull request template.
7. Engage in the discussion and address any feedback or questions raised during the code review process.
```

File: .github/CONTRIBUTIONS.md [hunk 2]
```diff
@@ -1,13 +1,58 @@
...
+
 ```bash
 ./lib/bashunit tests
 ```
+
+Please make sure to write appropriate tests for any new features or bug fixes you introduce. Tests should be placed in the `tests` directory and follow the naming convention of `test_*.sh`.
+
+## Contribution Process
+
+To contribute to this project, please follow these steps:
```
Comment (JavierCane) [inline .github/CONTRIBUTIONS.md:46]: Suggestion in order to explicitly state that we do not have any special requirement 😇

```suggestion
To contribute to this project, you can follow the common steps you would follow to contribute to any other Open Source project, that is:
```

**Review by JavierCane — APPROVED** [review]
Just 2 non-blocking suggestions 😊

File: .github/CONTRIBUTIONS.md (hunk 1 above)
Comment (johnlk) [reply .github/CONTRIBUTIONS.md:54]: yeah good call

**Pull request merged**